recipes/gimmemotifs/compile_fix.patch

```diff
@@ -0,0 +1,16 @@
+--- compile_externals.py	2019-07-16 12:48:20.464794208 +0200
++++ compile_externals.fix.py	2019-07-16 12:48:11.872510195 +0200
+@@ -10,11 +10,11 @@
+         return
+     
+     try:
+-        Popen("gcc", stdout=PIPE, stderr=PIPE).communicate()
++        Popen(os.environ["GCC"], stdout=PIPE, stderr=PIPE).communicate()
+     except Exception:
+         return
+     
+-    Popen(["gcc","-o%s" % name, "%s.c" % name, "-lm"], cwd=path, stdout=PIPE).communicate()
++    Popen([os.environ["GCC"],"-o%s" % name, "%s.c" % name, "-lm"], cwd=path, stdout=PIPE).communicate()
```

Comment:
I didn't see this before: You would want `CC` instead of `GCC` here to make it portable, i.e., to fix the macOS build.